pkg/asset/config/appliance_config.go

```diff
@@ -298,10 +307,6 @@ func (a *ApplianceConfig) validateConfig(f asset.FileFetcher) field.ErrorList {
 		}
 	}
 
-	// Validate CustomClusterManifests
-	if err := a.validateCustomClusterManifests(f); err != nil {
```

Comment:
proposal, which is probably out of scope for this PR: validate that the given YAML is in a valid YAML format.

Reply:
Please open a ticket for it so we won't forget

Reply:
Sure, will do.